build-system/tasks/make-extension/index.js

```diff
@@ -24,304 +24,161 @@ const {
 const {cyan, green, red, yellow} = require('kleur/colors');
 const {execOrThrow} = require('../../common/exec');
 const {log} = require('../../common/logging');
-const {makeBentoExtension} = require('./bento');
 
-const year = new Date().getFullYear();
-
-/*eslint "max-len": 0*/
 /**
- * @param {string} str
+ * Convert dash-case-name to PascalCaseName.
+ * @param {string} name
  * @return {string}
  */
-function pascalCase(str) {
-  return (
-    str[0].toUpperCase() +
-    str.slice(1).replace(/-([a-z])/g, function (g) {
-      return g[1].toUpperCase();
-    })
-  );
+function dashToPascalCase(name) {
+  return name.replace(/(?:-|^)([a-z])/g, (_, c) => c.toUpperCase());
 }
 
 /**
- * @param {string} name
- * @return {string}
+ * Create a mutator function from a map of keys/values to replace.
+ * @param {Object<string, string>} replaceMap
+ * @return {function(string): string}
  */
-function getValidatorFile(name) {
```

Comment:
All these template functions are gone, its file templates now in `template/classic`.